README.md

```diff
@@ -76,3 +76,959 @@ query.execute(conn, 'select distinct ?s where { ?s ?p ?o }', {
+### `db.optimize(conn, database, params)`
+
+Optimizes a database. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- database (`string`)
+
+- params (`Object`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+### `db.copy(conn, database, destination, params)`
+
+Makes a copy of a database. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- database (`string`)
+
+- destination (`string`)
+
+- params (`Object`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+### `db.list(conn, params)`
+
+Gets a list of all databases on a Stardog server. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- params (`Object`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+### `db.size(conn, database, params)`
+
+Gets number of triples in a database. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- database (`string`)
+
+- params (`Object`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+### `db.clear(conn, database, transactionId, params)`
+
+Clears the contents of a database. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- database (`string`)
+
+- transactionId (`string`)
+
+- params (`Object`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+### `db.namespaces(conn, database, params)`
+
+Gets a mapping of the namespaces used in a database. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- database (`string`)
+
+- params (`Object`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+### `db.exportData(conn, database, options, params)`
+
+Exports the contents of a database. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- database (`string`)
+
+- options ({ mimeType: [`AcceptMimeTypes`](#AcceptMimeTypes) })
+
+- params (`{ graphUri: string }`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+## options
+
+### `db.options.get(conn, database, params)`
+
+Gets set of options on a database. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- database (`string`)
+
+- params (`Object`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+### `db.options.set(conn, database, databaseOptions, params)`
+
+Sets options on a database. 
+
+Expects the following parameters:
+
+- conn ([`Connection`](#Connection))
+
+- database (`string`)
+
+- databaseOptions (`Object`)
+
+- params (`Object`)
+
+Returns [Promise<HTTP.Body>](#Body)
+
+## transaction
+
+### <a name="Encodings">Encodings</a>
+
+One of the following values:
+
+`'gzip' |
+                'compress' |
+                'deflate' |
+                'identity' |
+                'br'`
+### <a name="TransactionResponse">TransactionResponse</a>
+
+Object with the following values:
+
+- transactionId (`string`)
+
+### <a name="TransactionOptions">TransactionOptions</a>
+
+Object with the following values:
+
+- contentType (`HTTP.ContentMimeTypes`)
+- encoding (`Encodings`)
+
+### `db.transaction.begin(conn, database, params)`
```

Comment:
Let's do something to keep these headers consistent. Either use full anchor tags or the `###` not both.